src/App.js

```diff
@@ -0,0 +1,83 @@
+import React, { Component } from 'react';
+import Transactions from './components/Transactions';
+import Operations from './components/Operations';
+import { BrowserRouter as Router, Route, Link } from 'react-router-dom'
+import axios from 'axios'
+import './App.css';
+
+class App extends Component {
+  constructor(){
+    super()
+    this.state = {
+      data: [] 
+    }          
+  }
+
+  componentDidMount = async () =>{
+    await this.getData()
+  }
+
+  getData = async () =>{
+    let transactions = await axios.get("http://localhost:8000/transactions")
+    this.setState({ data: transactions.data }, function(){
+      console.log(this.state)
+    })
+  }
+
+  postData = async (transaction) => {
+    await axios.post('http://localhost:8000/transaction', transaction)
+  }
+
+  deleteFromDB = async(amount) => {
+    await axios.delete(`http://localhost:8000/transaction/${amount}`)
+  }
+
+  calculateBalance = () => {
+    let sum = 0
+    for(let data of this.state.data){
+      sum += parseInt(data.amount)
+    }
```

Comment:
check [this](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce) method